Compose update to 1.0.1

wiryadev opened the pull request:
- Kotlin update to 1.5.21
- Accompanist to 0.16.0
- Coil to 1.3.2
- More dependencies update
- replace deprecated observe from lifecycle

arriolac commented:
Hey @wiryadev, thanks for your contribution! Can you update your branch with the latest from the `compose` branch?

wiryadev commented:
> Hey @wiryadev, thanks for your contribution! Can you update your branch with the latest from the `compose` branch?

Hi, i have updated it with the latest `compose` branch and updated Coil with new `rememberAsycnImagePainter` they introduced in 2.x.x version.
